azavea/pwd-unitybar
# Show icons for nonempty search

## Overview

This PR fixes a bug in the Unity Bar whereby the map, help, and authentication action icons wouldn't display when the search box had a term entered in it. They'll still be hidden when the search box is expanded, but will return when the input loses focus.

Connects #70 
Connects https://github.com/azavea/pwd-parcel-viewer/issues/719

### Demo

![Screen Shot 2019-04-19 at 5 46 55 PM](https://user-images.githubusercontent.com/4165523/56445462-58404180-62cb-11e9-9b55-cfa35ab04ecc.png)

## Notes

First commit fixes part of an npm audit encountered issue with the tar package. We can't fully fix it yet because it's also a dependency of node-sass, which hasn't been updated since the issue was reported.

## Testing Instructions

- test the Netlify deployment and verify that see the action icons when there is a search term in the input but some other element has focus
